docs/dev/swagger.yaml

```diff
@@ -624,10 +624,13 @@ paths:
         **OJS 3.1 Target**:
           * See `/issues/{issueId}`.
           * Only published issues are returned.
-          * Only the `volume`, `number` and `year` parameters are supported.
```

Comment:
Would it be quick & easy to continue support for `volume`, `number`, and `year` single values?

Reply:
Good thinking.